### PR TITLE
i18n(zh-CN): update `components`

### PR DESCRIPTION
**Pull request by Muzych**
#### Description

- Closes # N/A
- What does this PR change? Give us a brief description. <br>Updates `components`

### CODE REVIEW

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 1]
```diff
@@ -295,3 +269,117 @@ import importedCode from '/src/env.d.ts?raw';
...
+- src
+  - components
+    - **Header.astro**
+  - …
+- pages/
+
+</FileTree>
+```
+
+以上代码生成结果如下：
```
Comment (liruifengv) [inline docs/src/content/docs/zh-cn/guides/components.mdx:304]:
```suggestion
以上代码在页面上生成了以下内容：
```
整体翻译保持统一吧。

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 2]
```diff
@@ -295,3 +269,117 @@ import importedCode from '/src/env.d.ts?raw';
 import importedCode from '/src/env.d.ts?raw';
 
 <Code code={importedCode} lang="ts" title="src/env.d.ts" />
+
+### 文件树
+
+我们可以通过`<FileTree>`组件来显示目录的结构，其中包括文件图标和可折叠的子目录。
+
+通过利用 `<FileTree>` 组件内置的 Markdown 无序列表语法，我们可以明确地指定文件和目录的组织结构。为了创建子目录，我们可以使用嵌套的列表项；若希望某个目录不显示具体内容，只需在列表项末尾添加斜杠即可。这样，我们可以清晰地呈现出目录的树状结构。
```
Comment (liruifengv) [inline docs/src/content/docs/zh-cn/guides/components.mdx:277]:
```suggestion
通过利用 `<FileTree>` 组件内置的 [Markdown 无序列表语法](https://www.markdownguide.org/basic-syntax/#unordered-lists)，我们可以明确地指定文件和目录的组织结构。为了创建子目录，我们可以使用嵌套的列表项；若希望某个目录不显示具体内容，只需在列表项末尾添加斜杠`/`即可。这样，我们可以清晰地呈现出目录的树状结构。
```
缺少链接

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 3]
```diff
@@ -295,3 +269,117 @@ import importedCode from '/src/env.d.ts?raw';
...
+   ```js
+   import { Steps } from '@astrojs/starlight/components';
+   ```
+
+2. 将有序列表放入该组件中
+
+</Steps>
+````
+
+以上代码生成结果如下：
```
Comment (liruifengv) [inline docs/src/content/docs/zh-cn/guides/components.mdx:344]:
```suggestion
以上代码在页面上生成了以下内容：
```

**Review by liruifengv — COMMENTED** [review]
Thanks for your update. @Muzych I left a little modification suggestion

**Comment by liruifengv** [comment]
Thank you @Muzych 

One thing I forgot to point out is that in the documentation, we don't recommend using words like `we` or `我们`. You can find some suggestions in the [Writing Style Guide](https://contribute.docs.astro.build/guides/writing-style/).

```
Do not use we, we’ll, us, let’s etc. (You are not with the reader.)
```

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 4]
```diff
@@ -295,3 +269,117 @@ import importedCode from '/src/env.d.ts?raw';
 import importedCode from '/src/env.d.ts?raw';
 
 <Code code={importedCode} lang="ts" title="src/env.d.ts" />
+
+### 文件树
+
+可以通过`<FileTree>`组件来显示目录的结构，其中包括文件图标和可折叠的子目录。
```
Comment (liruifengv) [inline docs/src/content/docs/zh-cn/guides/components.mdx:275]:
```suggestion
通过`<FileTree>`组件来显示目录的结构，其中包括文件图标和可折叠的子目录。
```

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 5]
```diff
@@ -295,3 +269,117 @@ import importedCode from '/src/env.d.ts?raw';
...
+
+通过利用 `<FileTree>` 组件内置的 [Markdown 无序列表语法](https://www.markdownguide.org/basic-syntax/#unordered-lists)，可以明确地指定文件和目录的组织结构。为了创建子目录，可以使用嵌套的列表项；若希望某个目录不显示具体内容，只需在列表项末尾添加斜杠`/`即可。这样，便可以清晰地呈现出目录的树状结构。
+
+可以使用以下语法自定义文件树的外观：
```
Comment (liruifengv) [inline docs/src/content/docs/zh-cn/guides/components.mdx:279]:
```suggestion
使用以下语法自定义文件树的外观：
```

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 6]
```diff
@@ -295,3 +269,117 @@ import importedCode from '/src/env.d.ts?raw';
...
+  - components
+    - **Header.astro**
+  - …
+- pages/
+
+</FileTree>
+
+### 步骤
+
+可以通过使用 `<Steps>` 组件为任务的编号列表设置样式。这对于需要清晰突出地显示每个步骤的分布指南很有用。
```
Comment (liruifengv) [inline docs/src/content/docs/zh-cn/guides/components.mdx:323]:
```suggestion
通过使用 `<Steps>` 组件为任务的编号列表设置样式。这对于需要清晰突出地显示每个步骤的分布指南很有用。
```

**Review by liruifengv — COMMENTED** [review]
No vague sentences

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 7]
```diff
@@ -295,3 +269,117 @@ import importedCode from '/src/env.d.ts?raw';
 import importedCode from '/src/env.d.ts?raw';
 
 <Code code={importedCode} lang="ts" title="src/env.d.ts" />
+
+### 文件树
+
+通过`<FileTree>`组件来显示目录的结构，其中包括文件图标和可折叠的子目录。
```
Comment (liruifengv) [inline docs/src/content/docs/zh-cn/guides/components.mdx:275]:
```suggestion
使用`<FileTree>`组件来显示带有文件图标和可折叠子目录的目录结构。
```

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 8]
```diff
@@ -295,3 +269,117 @@ import importedCode from '/src/env.d.ts?raw';
...
+
+通过利用 `<FileTree>` 组件内置的 [Markdown 无序列表语法](https://www.markdownguide.org/basic-syntax/#unordered-lists)，可以明确地指定文件和目录的组织结构。为了创建子目录，可以使用嵌套的列表项；若希望某个目录不显示具体内容，只需在列表项末尾添加斜杠`/`即可。这样，便可以清晰地呈现出目录的树状结构。
```
Comment (liruifengv) [inline docs/src/content/docs/zh-cn/guides/components.mdx:277]:
```suggestion
使用 `<FileTree>` 组件内置的 [Markdown 无序列表语法](https://www.markdownguide.org/basic-syntax/#unordered-lists)指定文件和目录的组织结构。使用嵌套的列表项创建子目录；若希望某个目录不显示具体内容，只需在列表项末尾添加斜杠`/`即可。
```

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 9]
```diff
@@ -295,3 +269,117 @@ import importedCode from '/src/env.d.ts?raw';
...
+
+使用以下语法自定义文件树的外观：
```
Comment (liruifengv) [inline docs/src/content/docs/zh-cn/guides/components.mdx:279]:
```suggestion
以下语法可用于自定义文件树的外观：
```

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 10]
```diff
@@ -295,3 +269,117 @@ import importedCode from '/src/env.d.ts?raw';
...
+  - components
+    - **Header.astro**
+  - …
+- pages/
+
+</FileTree>
+
+### 步骤
+
+通过使用 `<Steps>` 组件为任务的编号列表设置样式。这对于需要清晰突出地显示每个步骤的分布指南很有用。
```
Comment (liruifengv) [inline docs/src/content/docs/zh-cn/guides/components.mdx:323]:
```suggestion
使用 `<Steps>` 组件为任务的编号列表设置样式。这对于需要清晰突出地显示每个步骤的分布指南很有用。
```

**Review by liruifengv — COMMENTED** [review]
Make text more concise and clear

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 11]
```diff
@@ -295,3 +269,117 @@ import importedCode from '/src/env.d.ts?raw';
...
+- 使用 `...` 或 `…` 作为名称来添加占位符文件和目录。
+
+```mdx
+# src/content/docs/example.mdx
+
+import { FileTree } from '@astrojs/starlight/components';
+
+<FileTree>
+
+- astro.config.mjs an **important** file
```
Comment (HiDeoo) [inline docs/src/content/docs/zh-cn/guides/components.mdx:292]: The comment `an **important** file` can be safely translated. Would it make sense to do so?

I'll let you decide as you definitely know the best but if yes, the example below should also be updated.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for updating the translation :raised_hands: 

If I'm not mistaken, I think this may be missing the changes from https://github.com/withastro/starlight/pull/1568/files#diff-bde0eef3b5aa6eb8762ac30b4992c55505bc9eb0373f4a05d81f58f39c05e3bd
Would you be able to add the changes from that PR to this translation?

File: docs/src/content/docs/zh-cn/guides/components.mdx (hunk 11 above)
Comment (Muzych) [reply docs/src/content/docs/zh-cn/guides/components.mdx:292]: Yes, that makes sense, thank you for the advice, I will deal with these issues.

**Comment by Muzych** [comment]
> Hmm, looks like this PR is now empty after this [app](https://github.com/apps/pull) (I guess something installed on your end @Muzych ?) force-pushed some changes 🤔

I don't know how to deal with this situation, maybe I 'll open another pull request? I didn't realize that this app would have such a big impact on other projects.

**Comment by HiDeoo** [comment]
> I don't know how to deal with this situation, maybe I 'll open another pull request? I didn't realize that this app would have such a big impact on other projects.

No worries at all. If you still have the changes locally, for sanity, I would start by copying that file somewhere else just in case (if not, the changes are still available [here](https://github.com/withastro/starlight/compare/dec4d1843c647ff55e920d3d6522ad202009e32b..16e161411d4644cfcf137ebfe8018c3d5fcf6c7e#diff-1f70da17f969cf7f11852aa088dd94a7513af71b40853d812dcd378d63a9cb43)).

I guess I would also try to disable the app, at least for this repository, to avoid this happening again. I personally don't use it so no idea how it works.

Then, you should be able to just push a new commit with your previous changes and it should update the PR.

If you need more help or want me to do it for you, just let me know.

**Comment by Muzych** [comment]
> > I don't know how to deal with this situation, maybe I 'll open another pull request? I didn't realize that this app would have such a big impact on other projects.
> 
> No worries at all. If you still have the changes locally, for sanity, I would start by copying that file somewhere else just in case (if not, the changes are still available [here](https://github.com/withastro/starlight/compare/dec4d1843c647ff55e920d3d6522ad202009e32b..16e161411d4644cfcf137ebfe8018c3d5fcf6c7e#diff-1f70da17f969cf7f11852aa088dd94a7513af71b40853d812dcd378d63a9cb43)).
> 
> I guess I would also try to disable the app, at least for this repository, to avoid this happening again. I personally don't use it so no idea how it works.
> 
> Then, you should be able to just push a new commit with your previous changes and it should update the PR.
> 
> If you need more help or want me to do it for you, just let me know.

Ok, I 'll push previous changes again.

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 12]
```diff
@@ -58,23 +58,24 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 你可以使用 `<Tabs>` 和 `<TabItem>` 组件显示一个选项卡界面。
 每个 `<TabItem>` 必须有一个 `label` 来显示给用户。
+使用可选的`<icon>`属性在标签旁包含一个[Starlight内置图标](#所有图标)。
```
Comment (HiDeoo) [inline docs/src/content/docs/zh-cn/guides/components.mdx:61]:
```suggestion
使用可选的`icon`属性在标签旁包含一个[Starlight内置图标](#所有图标)。
```

One last suggestion regarding your last changes and I think we should be good to merge 🎉

Comment (Muzych) [reply docs/src/content/docs/zh-cn/guides/components.mdx]: Thanks for your patience.

**Review by HiDeoo — APPROVED** [review]
I think everything is in order now. Thanks for your patience, this was quite the ride but this is looking great now. :star2:

Thanks again for your contribution :raised_hands:

**Pull request merged**